spec/book_spec.rb

```diff
@@ -0,0 +1,10 @@
+require './book'
+describe Book do
+  let(:book) { Book.new('Title', 'Author') }
+  let(:person) { double('Person') }
+  let(:date) { '2023-10-19' }
+  it 'should initialize correctly' do
+    expect(book.title).to eq('Title')
+    expect(book.author).to eq('Author')
+  end
```

Comment:
- Kindly As it was stated in the requirements(Add unit tests for all the methods in all the classes:). We are expected to add unit test for the add_rental method for this classes as previous code reviwer seggested :+1: